pkg/convert/go.go

```diff
@@ -0,0 +1,69 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/message/pipeline"
+)
+
+func ToGo(m model.Messages) ([]byte, error) {
```

Comment:
Maybe there is a better way to name these functions?
 `ToGo()`  sounds a bit odd.

Reply:
We were using the same functions names in POC.

Reply:
I thought about that, maybe we can use convention `MessagesToGo` and `GoToMessages` and same for other languages `MessagesToNgx` and `NgxToMessages`, as I saw in other packages they are using some similar naming conventions for  convertions

Reply:
The narrative of the package is very narrow - only to serialize messages from/to different file formats. We won't transform other data structures in this package.

I believe we'll come up with a very good package name in the future which reflects the intent and narrative.

The package name is the culprit. I'm out of ideas at the moment.

Also, please note, we are not gonna make it right from the first go. We are open to changes.